src/chrome/main.html

```diff
@@ -1,92 +1,41 @@
 <!DOCTYPE html>
 <html lang="zh-TW">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>BreezeAds</title>
-    <style>
-        body {
-            font-family: Arial, sans-serif;
-        }
-
-        .container {
-            width: 300px;
-            margin: 0 auto;
-            border: 1px solid #ccc;
-            border-radius: 5px;
-            padding: 20px;
-        }
-
-        .header {
-            display: flex;
-            justify-content: space-between;
-            align-items: center;
-            border-bottom: 1px solid #ccc;
-            padding-bottom: 10px;
-            margin-bottom: 20px;
-        }
-
-        .header div {
-            display: flex;
-            align-items: center;
-        }
-
-        .header button {
-            background-color: transparent;
-            border: none;
-            cursor: pointer;
-        }
-
-        .content {
-            margin-bottom: 20px;
-        }
 
-        .content h3 {
-            margin-bottom: 10px;
-        }
-
-        .content p {
-            font-size: 12px;
-        }
-
-        .footer {
-            display: flex;
-            justify-content: space-between;
-            align-items: center;
-            font-size: 14px;
-            color: #888;
-        }
-
-        .footer a {
-            color: #000;
-            text-decoration: none;
-        }
-    </style>
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>BreezeAds</title>
+	<link href="https://unpkg.com/tailwindcss@^2.0/dist/tailwind.min.css" rel="stylesheet" />
```

Comment:
It can't use remote code (additional explanation needed). 
It may need to move to a local 'assets' folder.

Reply:
The local file has been added